clients/hadoopfs/pom.xml

```diff
@@ -287,7 +287,7 @@ To export to S3:
         <dependency>
             <groupId>io.lakefs</groupId>
             <artifactId>sdk</artifactId>
-            <version>1.18.0</version>
+            <version>1.53.1</version>
```

Comment:
Non blocking.  As always, I prefer big version bumps to occur in separate PRs.  Otherwise if things break, it is harder to tell if the breakage was due to the version bump or due to the change in functionality.